tidb-cloud/tidb-cloud-security-faq.md

```diff
@@ -0,0 +1,81 @@
+
+TiDB Cloud is mainly responsible for implementing and maintaining the sustainable, safe, stable, and compliant operation of TiDB Cloud.
+
+As a customer, you are your database owner, and you take the administrative and operational role of your database. You are primarily responsible for ensuring the security and compliance of your TiDB cluster within the TiDB Cloud. And your role in TiDB Cloud also involves implementing and managing any security features and access control policies of your own TiDB cluster.
+
+## Who can access customer data in TiDB Cloud?
+
+The security of customer data is of the highest priority to TiDB Cloud. Regardless of which TiDB Cloud region you have selected, TiDB Cloud implements rigorous technical, contractual, and organizational measures to safeguard your data confidentiality, integrity, and availability.
+
+PingCAP has passed the standard certification and verification of SOC2 Type II, General Data Protection Regulation (GDPR), Payment Card Industry Data Security Standard (PCI DSS), and Health Insurance Portability and Accountability Act of 1996 (HIPAA), and invites third-party auditors to audit the certified standards every year. For details, see [PingCAP Trust Center](https://www.pingcap.com/trust-compliance-center).
+
+In TiDB Cloud, the data access security measures includes the following:
+
+- TiDB Cloud has designed a multi-tenant TiDB Cloud database service platform that follows the multi-tenant isolation architecture and best practices of AWS and GCP SaaS applications. The security functions and isolation mechanisms include multi-tenant authentication and authorization, network access control, data encryption, and auditing, focusing on protecting customer data with ongoing security and compliance.
+
+- TiDB Cloud has designed a secure and trusted TiDB Cloud technical architecture. The architecture effectively isolates and prohibits any insiders from directly accessing the TiDB Cloud infrastructure. No PingCAP staff can directly access the TiDB Cloud infrastructure where customer data resides. If internal technical personnel needs to monitor and maintain the TiDB Cloud infrastructure, they must go through a strict approval process, obtain minimum permissions, and pass the Bastion audit to complete the approved operation and maintenance within a limited time. Moreover, all operational behaviors will be recorded and audited, and TiDB Cloud also configures monitoring and alarms for unauthorized operation and maintenance.
+
+- TiDB Cloud has designed an isolation structure for internal operation account permissions to prevent internal personnel from using a privileged service account to access TiDB clusters. The service account is only used for initializing Serverless Tier and Dedicated Tier through the TiDB Cloud service gateway, and PingCAP staff do not have access and permissions to this service account credentials. After TiDB Cloud is initialized, you have the maximum authority over your TiDB clusters.
```

Comment:
```suggestion
- TiDB Cloud has designed an isolation structure for internal operation account permissions to prevent internal personnel from using a privileged service account to access TiDB clusters. The service account is only used for initializing Serverless Tier and Dedicated Tier through the TiDB Cloud service gateway, and PingCAP staff do not have access and permissions to credentials of this service account. After TiDB Cloud is initialized, you have the maximum authority over your TiDB clusters.
```